lib/postgres_to_redshift.rb

```diff
@@ -83,28 +83,41 @@ def bucket
   def copy_table(table)
     tmpfile = Tempfile.new("psql2rs")
     zip = Zlib::GzipWriter.new(tmpfile)
+    chunksize = 5 * 1024 * 1024 * 1024
```

Comment:
Might be nice to throw these constants in the top of the file, and use them for readability:

```ruby
  KILOBYTE = 1024
  MEGABYTE = KILOBYTE * 1024
  GIGABYTE = MEGABYTE * 1024
```